packages/eslint-config-airbnb/test/test-react-order.js

```diff
@@ -24,7 +24,7 @@ function lint(text) {
 }
 
 function wrapComponent(body) {
-  return `
+  return `\
```

Comment:
```suggestion
  return `
```
template literals don't need a backslash here.